app/controllers/comments_controller.rb

```diff
@@ -1,13 +1,20 @@
 class CommentsController < ApplicationController
   load_and_authorize_resource
   before_action :set_user_and_post
+
+  def index
+    @comment = Comment.new
+    render json: @comment, only: [:Text]
+  end
+
```

Comment:
- [ ] Kindly fix the error preventing the creation of a comment as well via  API.
[cinnamon-2023-12-23T111217+0200.webm](https://github.com/ismayito/Rails-blog-app/assets/81932478/4f8d0e07-211d-4f92-af9a-fcff65ab7ed8)